core/assets/bundles/bundle_nl.properties

```diff
@@ -118,619 +134,654 @@ players = {0} Spelers online
-zone.overgrowth.description = This area is overgrown, closer to the source of the spores.\nThe enemy has established an outpost here. Build dagger units. Destroy it. Reclaim that which was lost.
-zone.tarFields.description = The outskirts of an oil production zone, between the mountains and desert. One of the few areas with usable tar reserves.\nAlthough abandoned, this area has some dangerous enemy forces nearby. Do not underestimate them.\n\n[lightgray]Research oil processing technology if possible.
-zone.desolateRift.description = An extremely dangerous zone. Plentiful resources, but little space. High risk of destruction. Leave as soon as possible. Do not be fooled by the long spacing between enemy attacks.
-zone.nuclearComplex.description = A former facility for the production and processing of thorium, reduced to ruins.\n[lightgray]Research the thorium and its many uses.\n\nThe enemy is present here in great numbers, constantly scouting for attackers.
-zone.fungalPass.description = A transition area between high mountains and lower, spore-ridden lands. A small enemy reconnaissance base is located here.\nDestroy it.\nUse Dagger and Crawler units. Take out the two cores.
+zone.fungalPass.name = Schimmelpad
+zone.groundZero.description = De optimale plek om weer tot kracht te komen. Weinig gevaar. Weinig grondstoffen.\nVerzamel zoveel mogelijk lood en koper als je kan.\nEn vertrek.
+zone.frozenForest.description = Ook hier, dicht bij de bergen, hebben de schrimmels zich verspreid. De koude tempratuur houd ze niet voor eeuwig tegen.\n\nBegin de industriele revolutie. Bouw fossiele generators. Leer hoe te repareren.
+zone.desertWastes.description = Deze woestijn is groot, onvoorspelbaar, en vol met oude technologie.\nSteenkool is hier te vinden. Verbrand het om stroom op te wekken, or verwerk het in grafiet.\n\n[lightgray]Of het hier veilig is is een tweede.
+zone.saltFlats.description = Aan de randen van de woestijn liggen de zoutvlaktes. Weinig grondstoffen zijn hier te vinden.\n\nDe vijand heeft hier rantsoenen opgeslagen. Vernietig hun core. Laat niks staan.
+zone.craters.description = Water heeft zich hier opgehoopt, herrinering aan de vroegere oorlog. Herrover dit gebied. Verzamel zand. Maak glas. Pomp water in je wapens en boren.
+zone.ruinousShores.description = Voorbij de ruines, is de kust. Lang geleden werd de kust hier verdedigd. Weinig ervan is terug te vinden. Enkel de meest simpele verdedigingswerken staan nog overeind, \nGa door met uitbereiden, herontdek de verloren techniek.
+zone.stainedMountains.description = Verder inland liggen de bergen, nog niet berijkt door schimmels.\nVerzamel de grote hoeveelheiden titanium in het gebied. Leer hoe je het kan gebruiken.\n\nDe vijand is krachtig hier. Geef ze geen tijd om je te overrompelen.
+zone.overgrowth.description = Dit gebied is overgroeid, dichter bij de bron van de schimmels.\nDe vijand heeft hier een uitkijkpost. Bouw dolk units. Vernietig hen. Claim was verloren was.
```

Comment:
`hen` Who?
(This problem also exists in the source string but because English is more flexible on transitivity rules it can be left as-is there. Dutch transitivity rules do not allow this.)

The last section of this string needs work.
Suggestion: `Hergebruik wat ooit verloren was.` **This also needs work!**